app/_src/policies/meshidentity.md

```diff
@@ -0,0 +1,216 @@
+
+```yaml
+spec:
+  selector:
+    dataplane:
+      matchLabels:
+        app: my-app
+```
+
+##### Apply to nothing
+
+```yaml
+spec:
+  selector: {}
+```
+
+### SpiffeID
+
+The `spiffeID` field lets you override how SPIFFE IDs are constructed for the data plane proxies selected by this `MeshIdentity`.
+By default, {{site.mesh_product_name}} generates a SPIFFE ID based on the mesh and zone.
+With `spiffeID`, you can customize the `trustDomain` and the `path` template.
+
+{% raw %}
+```yaml
+spec:
+  spiffeID:
+    trustDomain: "{{ .Mesh }}.{{ .Zone }}.mesh.local"
+    path: "/ns/{{ .Namespace }}/sa/{{ .ServiceAccount }}"
+```
+{% endraw %}
+
+Supported variables in `trustDomain` field are:
+* `.Mesh`
+* `.Zone`
+
+Supported variables in `path` field are:
+* `.Namespace`
+* `.ServiceAccount`
+
+Also, both in `trustDomain` and `path` it's possible to use resource's `labels`, i.e.:
+
+{% raw %}
+```yaml
+spec:
+  spiffeID:
+    trustDomain: '{{ label "kuma.io/mesh" }}.{{ label "kuma.io/zone" }}.mesh.local'
+    path: '/ns/{{ label "k8s.kuma.io/namespace" }}/sa/{{ label "k8s.kuma.io/service-account" }}'
+```
+{% endraw %}
+
+### Provider
+
+The `provider` field defines how identity certificates are issued.
+This field is required and must specify one of the supported provider types:
+
+* `Bundled` – certificates are issued by {{site.mesh_product_name}}’s control plane, either autogenerated or supplied by the user.
+* `Spire` – certificates are issued directly by a SPIRE Agent through SDS.
+
+## Examples
+
+### Minimal `Bundled` MeshIdentity
+
+{% policy_yaml %}
+{% raw %}
+```yaml
+type: MeshIdentity
+name: identity
+mesh: default
+spec:
+  selector:
+    dataplane:
+      matchLabels: {} # apply to all data plane proxies in the mesh
+  provider:
+    type: Bundled
+    bundled:
+      meshTrustCreation: Disabled # do not automatically create a MeshTrust from this identity
+      insecureAllowSelfSigned: true # explicitly allow use of a self-signed CA
+      autogenerate:
+        enabled: true # let the control plane autogenerate a CA and store it
+```
+{% endraw %}
+{% endpolicy_yaml %}
+
+### MeshIdentity with CA provided by user
+
+{% policy_yaml %}
+{% raw %}
+```yaml
+type: MeshIdentity
+name: identity
+mesh: default
+spec:
+  selector:
+    dataplane:
+      matchLabels:
+        app: my-app # apply to all data plane proxies with label `app: my-app`
+  spiffeID:
+    trustDomain: "{{ .Mesh }}.{{ .Zone }}.mesh.local"
+    path: "/ns/{{ .Namespace }}/sa/{{ .ServiceAccount }}"
+  provider:
+    type: Bundled
+    bundled:
+      meshTrustCreation: Enabled # automatically create a MeshTrust from this identity
+      insecureAllowSelfSigned: true # explicitly allow use of a self-signed CA
+    ca:
+      certificate:
+        type: File
+        file:
+          path: /ca.crt
+      privateKey:
+        type: File
+        file:
+          path: /ca.key
+```
+{% endraw %}
+{% endpolicy_yaml %}
+
+### MeshIdentity with `Spire` provider
+
```

Comment:
Missing additional informations:
You need to enable injection of spire socket to the pod https://github.com/kumahq/kuma/blob/master/pkg/config/plugins/runtime/k8s/config.go#L106-L109 or user can label just one pod `k8s.kuma.io/spire-support`